packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx

```diff
@@ -173,19 +173,44 @@ describe('mapProjectToApplicationContextProject', () => {
 });
 
 describe('mapEnvironmentToApplicationContextEnvironment', () => {
-  it('should map environment to environment context', () => {
-    expect(
-      mapEnvironmentToApplicationContextEnvironment(createTestEnvironment())
-    ).toEqual({
-      revision: expect.any(String),
-      applicationName: expect.any(String),
-      frontendHost: expect.any(String),
-      mcApiUrl: 'https://mc-api.commercetools.com',
-      location: expect.any(String),
-      env: expect.any(String),
-      cdnUrl: expect.any(String),
-      servedByProxy: expect.any(Boolean),
-      foo: expect.any(String),
+  describe('when application is configured to run behind proxy', () => {
```

Comment:
I felt this kind of narrow "integration" test is needed. The other permutations are unit tested in the the module.